addon/components/flexberry-menu.js

```diff
@@ -255,5 +260,21 @@ export default FlexberryBaseComponent.extend({
   */
   _getActionForMenu(collapseMenuOnItemClick) {
     return collapseMenuOnItemClick ? 'activate' : 'nothing';
+  },
+
+  actions: {
+    /**
+      Handler to get user button's actions and send action to corresponding controllers's handler.
+      @method actions.customButtonAction
+      @public
```

Comment:
По умолчанию все публичное, нужно только для приватных тег добавлять.
И параметр `item` описать.
```suggestion

      @method actions.customButtonAction
```